lib/install.js

```diff
@@ -213,7 +215,31 @@ function install(opts, cb) {
     );
     fs.createReadStream(gzipFilePath).pipe(gunzip).pipe(writeStream)
       .once('error', cb.bind(null, new Error('Could not read ' + gzipFilePath)))
-      .once('finish', function() { cb() })
+      .once('finish', function() {
```

Comment:
Should'nt we check in advance that it's a tar.gz and avoid trying to ungzip etc?